test/unit/mocha-globals.js

```diff
@@ -15,7 +15,7 @@ chai.use(require('sinon-chai'))
 chai.use(require('chai-subset'))
 
 beforeEach(() => {
-  global.sinon = sinon.sandbox.create()
+  global.sinon = sinon.createSandbox()
```

Comment:
AFAIK sinon is now sandbox by default, so it should be ok to remove this line

Reply:
The sinon docs seem to agree with you, but removing these lines causes the tests to fail in some mysterious way.  The sinon docs https://sinonjs.org/guides/migrating-to-5.0 say
```
The old sandbox API is still available, so you don’t have to do anything.
```
so I propose to go with that for now.